README.md

```diff
@@ -846,4 +846,4 @@ Thanks to [derhuerst](https://github.com/derhuerst), creator of prompt libraries
 ### License
 
 Copyright © 2018, [Jon Schlinkert](https://github.com/jonschlinkert).
-Released under the [MIT License](LICENSE).
\ No newline at end of file
+Released under the [MIT License](LICENSE).
```

Comment:
I made this change in the github online editor, not sure why this line was affected.

Reply:
Seems like there was no newline at the end of the file, but normally I'd expect Github to render that.